m/mathutil/mathutil_ubi8.5.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : mathutil
+# Version       : v1.0.0
+# Source repo   : https://gitlab.com/cznic/mathutil
+# Tested on     : ubi 8.5
+# Language      : GO
+# Travis-Check  : True
```

Comment:
Looks like, Test execution is taking more than 40 mins.
Can you please run the tests on VM and update the status in comments at the end of the script ?